tools/source_updater/updater/handler.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import subprocess
+from typing import Set
+
+from airbyte_cdk.sources.declarative.manifest_declarative_source import ManifestDeclarativeSource
+from updater.catalog import CatalogMerger
+from updater.config import Config
+from updater.source import SourceRepository
+
+
+logger = logging.getLogger("handler")
+
+
+class SourceUpdaterHandler:
+    def __init__(self, source_repository: SourceRepository, catalog_merger: CatalogMerger):
+        self._source_repository = source_repository
+        self._catalog_merger = catalog_merger
+
+    def handle(self, source_name: str, new_manifest_source: ManifestDeclarativeSource, main_config: Config, other_configs: Set[Config]):
+        # FIXME eventually extract git related command somewhere it makes sense
+        # FIXME is there a state of the local git we would like to enforce?
+        branch_name = f"source-updater/updating-{source_name}"
```

Comment:
presumably will be a command line argument?

Reply:
This is a good question. What are the concerns? I know there is a concern of clashes there.

I would like to avoid exposing this to the user. If the goal is to have SE use this or to be part of the Connector Builder, I don't think the user needs to know about that.